docs/source/Advanced-Usage.md

```diff
@@ -158,9 +158,16 @@ In the `env_file_app_template`, you'd see various elasticsearch related environm
 #### Kibana
 Intel Owl provides a Kibana's "Saved Object" configuration (with example dashboard and visualizations). It can be downloaded from [here](https://github.com/intelowlproject/IntelOwl/blob/develop/configuration/Kibana-Saved-Conf.ndjson) and can be imported into Kibana by going to the "Saved Objects" panel (http://localhost:5601/app/management/kibana/objects).
 
+#### Spin up local container for development
+1. Set ```LOCAL_ELASTICSEARCH=1``` in ```docker/.env.start.test```.
```

Comment:
this documentation is not correct. As already said (this is the third time I write this),  we do not need this for development purposes. The use case is different from GreedyBear.

This should be an alternative of the step 1 in `Example Configuration`: either you connect to an external ElasticSearch (this remains the preferred way) or you use the new additional configuration to run it together with IntelOwl (but that is really a bad configuration because the required resources for the system would be really too much and we must say this)